giskard_cicd/automation/post_discussion.py

```diff
@@ -28,12 +28,12 @@ def construct_opening(dataset_id, dataset_config, dataset_split, vulnerability_c
 
 def construct_closing(test_suite_url=None):
     giskard_hub_wording = f"""
-    \n\nWe've generated test suites according to your scan results! Checkout the [Test Suite in our Giskard Space]({test_suite_url})
+    \n\nWe've generated test suites according to your scan results! Checkout the [Test Suite in our Giskard Space]({test_suite_url}) and [Giskard Documentation](https://docs.giskard.ai/en/latest/getting_started/quickstart/index.html) to learn more about how to test your model.
```

Comment:
Is it better to directly use https://docs.giskard.ai/en/stable/getting_started/quickstart/quickstart_nlp.html for text classification models?